build-logic/src/main/kotlin/com/ibm/wala/gradle/JavaCompileUsingEcj.kt

```diff
@@ -65,10 +65,18 @@ abstract class JavaCompileUsingEcj : JavaCompile() {
 
   @TaskAction
   protected override fun compile(inputs: InputChanges) {
+    val testArgs = options.allCompilerArgs
+    val argFile = createTempFile(prefix = "kotlinTemp", suffix = ".tmp")
+    val writer = java.io.PrintWriter(argFile)
+    for (testArg in testArgs) {
+      writer.print(testArg + " ")
+    }
+    writer.close()
```

Comment:
Use Kotlin's `use { ... }` extension on the `PrintWriter` instance instead of an explicit `close()`.

Reply:
The revised code's use of `use { ... }` looks good. Thanks!